web/static/js/components/timezones/TimezoneAutocomplete.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { PropTypes, Component } from 'react'
+import { fetchTimezones } from '../../actions/timezones'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import { Autocomplete } from '../ui'
+import * as actions from '../../actions/survey'
+import { formatTimezone } from './util'
+import 'materialize-autocomplete'
+
+class TimezoneAutocomplete extends Component {
+  static propTypes = {
+    t: PropTypes.func,
+    dispatch: PropTypes.func.isRequired,
+    selectedTz: PropTypes.string,
+    timezones: PropTypes.object
+  }
+
+  constructor(props) {
+    super(props)
+    this.timeZoneInput = null
+    this.state = {
+      fetching: false
+    }
+  }
+
+  setTimezoneInputVal = (timezone) => {
+    if (this.timeZoneInput) this.timeZoneInput.value = timezone
+  }
+
+  focusTimezoneInput = () => {
+    if (this.timeZoneInput) this.timeZoneInput.focus()
+  }
+
+  getTimezoneInputRef = () => {
+    return this.timeZoneInput
+  }
+
+  componentDidUpdate() {
+    const { selectedTz } = this.props
+    this.focusTimezoneInput()
```

Comment:
Setting the focus on the component is making the page scroll down to the timezone autocomplete as soon as the New Survey page loads. This is definitely not the desired behaviour.

Is setting the focus needed at all?